react/components/AutocompleteResults/index.js

```diff
@@ -36,10 +36,10 @@ const getLinkProps = element => {
   if (element.criteria) {
     // This param is only useful to track terms searched
     // See: https://support.google.com/analytics/answer/1012264
```

Comment:
Make it clear that the param is the `_q`
```suggestion
    // This param _q is only useful to track terms searched
    // See: https://support.google.com/analytics/answer/1012264
    // And: https://vtex.io/docs/recipes/store-management/setting-up-google-analytics-search-tracking/
```